lib/core/presentation/widgets/poap/hot_badge_item.dart

```diff
@@ -0,0 +1,263 @@
+    }
+
+    const startAngle = 0.6 + pi;
+    final sweepAngle = (pi - 1.2) * progress;
+    canvas.drawArc(rect, startAngle, sweepAngle, false, paint);
+  }
+
+  @override
+  bool shouldRepaint(covariant CustomPainter oldDelegate) {
+    return true;
+  }
+}
+
+class _BadgeLocationTag extends StatelessWidget {
+  const _BadgeLocationTag();
+
+  @override
+  Widget build(BuildContext context) {
+    final colorScheme = Theme.of(context).colorScheme;
+    return Container(
+      padding: EdgeInsets.all(Spacing.superExtraSmall),
+      decoration: ShapeDecoration(
+        color: colorScheme.primary.withOpacity(0.3),
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(
+            LemonRadius.extraSmall,
+          ),
+        ),
+      ),
+      child: ClipRRect(
+        child: BackdropFilter(
+          filter: ImageFilter.blur(sigmaX: 3, sigmaY: 3),
+          child: Container(
+            child: Row(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                ThemeSvgIcon(
+                  color: colorScheme.onPrimary,
+                  builder: (filter) => Assets.icons.icNavigationFilled.svg(colorFilter: filter),
+                ),
+                SizedBox(width: Spacing.superExtraSmall / 2),
+                Text(
+                  '1.2km',
+                  style: Typo.xSmall.copyWith(
+                    color: colorScheme.onPrimary,
+                    fontWeight: FontWeight.w600,
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class _BadgeName extends StatelessWidget {
+  const _BadgeName();
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.symmetric(
+        vertical: Spacing.extraSmall,
+      ),
+      child: Column(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Text(
+            'Raging Burger',
+            maxLines: 1,
+            overflow: TextOverflow.ellipsis,
+            style: Typo.small.copyWith(
+              fontWeight: FontWeight.w600,
+            ),
+          ),
+          SizedBox(height: 2.w),
+          Text(
+            '82/100 claimed',
+            style: Typo.xSmall.copyWith(
+              fontWeight: FontWeight.w400,
+              color: Theme.of(context).colorScheme.onSecondary,
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+const collectionThumbnails = [
+  'https://s3-alpha-sig.figma.com/img/b2b5/81b9/060fc740efe7a7c28ec3b2acd4147df2?Expires=1693785600&Signature=TGMwFv-Tke6mJzC70ESjbSGLEdK99fjgYEpp6janm8RsKyVLZJOOmBRwP9~m8DLJycz8q7hkEIevkOumQ8MkU910Rblt2efcd4pmexwyWdeMgb7eLz748W4tw7X5uN0Lh0zUF720Kzs7LogYosiIz-FXAzskCL9K8aZ9pXspjKaNymugET6mGPLQ78doux5m~VbBy2Mjf293rTyUD88VeIJ2hBxYLsOMCiaVoeJF6lB4VQBpPlYES~ljRY~WXeYxxV1i7TdLql3JylEEqCjirONolstSycm6rgc8SCSDLFmTmTxY6~0mwvs3qAL9lhQ8mahn3kb3dJOvgHbVklPdUA__&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4',
```

Comment:
This is a good example of mock data, keep it up!

Reply:
You can put it all these mock value in 1 file so we can just delete it all at once when we no longer need it

Reply:
I will removed when implement integration with BE,  dont wanna create a separated file